CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ Main (unreleased)
 
 ### Features
 
+- Add `structured_metadata_drop` stage for `loki.process` to filter structured metadata. (@baurmatt)
```

Comment:
We just cut a rc release, could you merge main and move this entry so it does not end up under v1.11.0-rc.0=